web-ui/src/main/resources/catalog/components/edit/onlinesrc/OnlineSrcDirective.js

```diff
@@ -1236,9 +1236,9 @@
                            function(l) {
                              if (angular.isDefined(l.name)) {
                                scope.layers.push({
-                                 Name: l.name.localPart,
-                                 abstract: l._abstract,
-                                 Title: l.title
+                                 Name: l.name.prefix+':'+l.name.localPart,
+                                 abstract: angular.isArray(l._abstract)?l._abstract[0].value:l._abstract,
+                                 Title: angular.isArray(l.title)?l.title[0].value:l.title
```

Comment:
The logic you added here works, but is missing in the viewer "add layer" panel, so some WFS2 services cannot be added to the map:
http://services.inspire-provincies.nl/ProductionAndIndustrialFacilities/services/download_PF?request=getcapabilities&service=wfs

This backs your idea of providing a version-agnostic capabilities document.

Reply:
i havan't touched adding wfs2 to the map, i suggest to move that to a new PR, since it may be a big development?

Reply:
True, there may be some other fixes needed to make it work.